utils/coverage_report.sh

```
@@ -1,3 +1,8 @@
+if [ -d build ]; then
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)